hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestInsertTable.scala

```diff
@@ -2075,6 +2075,45 @@ class TestInsertTable extends HoodieSparkSqlTestBase {
     })
   }
 
+  test("Test vectorized read nested columns for LegacyHoodieParquetFileFormat") {
+    withSQLConf(
+      "hoodie.datasource.read.use.new.parquet.file.format" -> "false",
+      "hoodie.file.group.reader.enabled" -> "false",
+      "spark.sql.parquet.enableNestedColumnVectorizedReader" -> "true",
+      "spark.sql.parquet.enableVectorizedReader" -> "true") {
```

Comment:
will this test cover all the spark releases above 3.3.0 ?

Reply:
Hmm, saw some Travis failures.

Reply:
Yes, this is caused by my modification. I'm trying to fix them.